### PR TITLE
Feature/use old twill embedded kafka server for kafka7

### PR DESCRIPTION
**Pull request by gsps1**
fixing unit tests failure.

### CODE REVIEW

File: cdap-kafka-pack/cdap-kafka-flow/pom.xml [hunk 1]
```diff
@@ -120,11 +120,6 @@
         </exclusions>
       </dependency>
       <dependency>
-        <groupId>org.slf4j</groupId>
```
Comment (anwar6953) [inline cdap-kafka-pack/cdap-kafka-flow/pom.xml]: instead, make this test scope

File: cdap-kafka-pack/cdap-kafka-flow/cdap-kafka-flow-core/pom.xml [hunk 2]
```diff
@@ -48,10 +48,6 @@
       <groupId>org.apache.zookeeper</groupId>
       <artifactId>zookeeper</artifactId>
     </dependency>
-    <dependency>
```
Comment (anwar6953) [inline cdap-kafka-pack/cdap-kafka-flow/cdap-kafka-flow-core/pom.xml]: instead, make this test scope (and don't remove it from outer pom's dependencyManagement)

**Comment by gsps1** [comment]
@anwar6953 addressed the comments.

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**